src/portfolio.js

```diff
@@ -118,7 +118,8 @@ To know how to get github key look at readme.md */
 
 const openSource = {
   githubConvertedToken: "Your Github Converted Token",
-  githubUserName: "Your Github Username"
+  githubUserName: "Your Github Username",
+  showProfile:"true" // true or false default false
```

Comment:
Can we change it to something like ``showGithubProfile`` too long but I think mentioning this would help out the user.